patches/tModLoader/Terraria/GameContent/ItemDropRules/ItemDropRule.cs.patch

```diff
@@ -0,0 +1,11 @@
+--- src/Terraria/Terraria/GameContent/ItemDropRules/ItemDropRule.cs
++++ src/tModLoader/Terraria/GameContent/ItemDropRules/ItemDropRule.cs
+@@ -2,7 +_,7 @@
+ {
+ 	public class ItemDropRule
+ 	{
+-		public static IItemDropRule Common(int itemId, int dropsOutOfX = 1, int minimumDropped = 1, int maximumDropped = 1) => new CommonDrop(itemId, dropsOutOfX, minimumDropped, maximumDropped);
++		public static IItemDropRule Common(int itemId, int dropsOutOfX = 1, int minimumDropped = 1, int maximumDropped = 1, int dropsXoutofY = 1) => new CommonDrop(itemId, dropsOutOfX, minimumDropped, maximumDropped, dropsXoutofY);
```

Comment:
Need an example using the `dropsXoutofY` parameter, and a description of how it works.

Reply:
Also not sure if vanilla has flubbed here, but the parameter to `Common` `dropsOutOfX` is called `dropsOutOfY` in `CommonDrop` looks like vanilla wants nice integer probabilities like 1/Y, and you're exposing the extra parameter for things like 2/7

Reply:
Its kinda odd. Vanilla is contradictory with how it uses this. I definitely need to do more testing with it once I do ExampleMod stuff.